### PR TITLE
[FxCop] roslyn-analyzers-runner provides a static code analysis without running build

### PR DESCRIPTION
**Pull request by suetsuguk**
Related  to sider/runners#971

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ This section has documents about analysis tools which Sider supports. We have an
 - [Cppcheck](./tools/cplusplus/cppcheck.md)
 - [cpplint](./tools/cplusplus/cpplint.md)
 
-### C
+### C\#
```
Comment (ybiquitous) [inline docs/index.md:90]: 👍

File: docs/tools/csharp/fxcop.md [hunk 2]
```diff
@@ -1,37 +1,20 @@
 ---
 id: fxcop
 title: FxCop
-sidebar_label: FxCop (beta)
+sidebar_label: FxCop
```
Comment (ybiquitous) [inline docs/tools/csharp/fxcop.md:4]: Please revert this change. We want to provide the beta support yet.

File: docs/tools/csharp/fxcop.md [hunk 3]
```diff
@@ -1,37 +1,20 @@
...
 hide_title: true
 ---
 
 # FxCop
 
-> This is **BETA**. The behavior of this tool might change.
-
```
Comment (ybiquitous) [inline docs/tools/csharp/fxcop.md:11]: Same here.

File: docs/tools/csharp/fxcop.md [hunk 4]
```diff
@@ -1,37 +1,20 @@
...
 | Supported Version | Language | Website                                                                    |
 | :---------------- | :------- | :------------------------------------------------------------------------- |
 | 2.9.8             | C#       | https://docs.microsoft.com/en-us/visualstudio/code-quality/fxcop-analyzers |
 
-**FxCop** is a static analysis tool for [.NET platform](https://dotnet.microsoft.com/). It supports both of C# and Visual Basic .NET. However, Sider supports C# project only now.
```
Comment (ybiquitous) [inline docs/tools/csharp/fxcop.md:16]: [IMO] I prefer that a brief description of the tool is put on the top of the document.
Because I think many users don't know what tool FxCop is.

File: docs/tools/csharp/fxcop.md (hunk 2 above)
Comment (suetsuguk) [reply docs/tools/csharp/fxcop.md]: LGTM. reverted the change.

File: docs/tools/csharp/fxcop.md (hunk 3 above)
Comment (suetsuguk) [reply docs/tools/csharp/fxcop.md]: Also reverted.

File: docs/tools/csharp/fxcop.md (hunk 4 above)
Comment (suetsuguk) [reply docs/tools/csharp/fxcop.md]: Agreed. I have moved this portion to the original position. Now the difference between master and this version is only a few changes.

**Comment by ybiquitous** [comment]
I will merge this when the runner is published.

**Comment by suetsuguk** [comment]
Thank you in advance.
